apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-content.js

```diff
@@ -74,10 +78,27 @@ function getTourContent() {
 			animation: 'undo-button',
 		},
 		{
-			heading: __( 'Congratulations!', 'full-site-editing' ),
-			description: __(
-				'You’ve now learned the basics. Remember, your site is always private until you decide to launch.',
-				'full-site-editing'
+			heading: __( 'Congratulations!', 'full-site-editing' ).replace( '&nbsp;', ' ' ),
```

Comment:
If you want to be real fancy you can replace with a real non-breaking space so that it wouldn't cause a line wrap (which is presumably the intention in French)

```suggestion
			heading: __( 'Congratulations!', 'full-site-editing' ).replace( '&nbsp;', '\u00A0' ),
```

Reply:
We've fixed this in the translation itself (with a real non-breaking space there), so we can drop this `replace` now.

Reply:
Oh, well that's even better!